pkg/clients/tsclient/client.go

```diff
@@ -35,6 +35,7 @@ import (
 
 const (
 	binaryContentType = "application/timeseries-binary-0"
+	jsonContentType   = "application/json"
```

Comment:
why are headers not already a const in the http package?

Reply:
I think there's a proposal but it was never merged